api/Headers.json

```diff
@@ -419,7 +419,7 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```

Comment:
This prompted a little digging on my part. This change is correct, but I'll be opening a PR to remove this feature entirely. 🎉